contracts/epochs/FlowEpoch.cdc

```diff
@@ -494,6 +831,7 @@ access(all) contract FlowEpoch {
 
             // Calculate rewards for the current epoch
             // and set the payout for the next epoch
+            // TODO(jord): do we need this? L862 we calculate rewards on enter EpochSetup phase
```

Comment:
I think we need the `FlowEpoch.calculateAndSetRewards()` in case the pre-spork epoch was ended in the Staking auction phase. If the pre-spork epoch was ended in the Setup/Committed phase, then rewards would have already been calculated.